### DISCUSSION TITLE
Error In community_detection_showcase.Rmd Due To Incorrect Perceval or Git Repo Path

### DISCUSSION THREAD

**Discussion by anthonyjlau** [sailuh/kaiaulu#266] [Q&A] [answered]
I get an error when trying to run community_detection_showcase.Rmd that says it is unable to generate git logs. I suspect that this is because my Perceval or git repo path is wrong because there is an error when trying to run parse_gitlog(perceval_path, git_repo_path). 

I am unable to run **which perceval** to find the path for reasons I am not sure about. However, I used the path /home/anthony/.local/bin/perceval in the tools.yml file. 

The git_repo_path is the default one set in apr.yml: ../../rawdata.git_repo/APR/.git

I was wondering if my paths are correct? And if so is there something that I may be missing?

**Comment by carlosparadis** [comment]
Possibly related to https://github.com/sailuh/kaiaulu/discussions/237 if the issue is Perceval, or 

https://github.com/sailuh/kaiaulu/discussions/265 if the issue is not locating the data to be parsed.Were you able to run the Git Log showcase Notebook? That also requires Perceval.

**Reply by anthonyjlau** [reply]
I am unable to run the gitlog_showcase.Rmd notebook. I also get an error when trying to run parse_gitlog.

**Reply by carlosparadis** [reply]
@anthonyjlau Could you post another question for the gitlog_showcase.Rmd Notebook and reference here? Please let me know on the other question (not this one) the error you experience there. It may be easier to start from that Notebook.

**Reply by anthonyjlau** [reply, marked as answer]
@carlosparadis The error I was getting for gitlog_showcase was the same as #265. To solve my problem, I manually created the git_repo folder in my rawdata folder. Then, inside of the git_repo folder, I used git clone on the apr repository link.
